Content.Server/Ghost/GhostSystem.cs

```diff
@@ -265,5 +267,11 @@ private IEnumerable<string> GetLocationNames()
 
             return players;
         }
+
+        public void OnStorageInsertAttempt(EntityUid uid, GhostComponent comp, InsertIntoStorageAttemptEvent args)
+        {
+            if (TryComp<GhostComponent>(uid, out var ghostcomp))
+                args.Cancel();
```

Comment:
```suggestion
            args.Cancel();
```

not sure if this one is even really necessary. they aren't items

Reply:
The check is necessary, i.e crates eat ghosts if this doesn't exist :3
But ye, I can remove the check for the component and it works a-ok!